### ISSUE TITLE
onAnimationEnd and children

### ISSUE THREAD

**Issue by pryahin** [Stanko/react-animate-height#61]
Hello, I found a strange behavior on the onAnimationEnd event.
If an element has a child that has an animation, the onAnimationEnd event will be triggered after the animation of the child has completed.

I think that onAnimationEnd should only work for AnimateHeight element, and should not be triggered on children's animations.

**Comment by Stanko** [comment]
Hello @pryahin,
I can't replicate what you are describing, check the quick example I made:
https://codesandbox.io/s/241mpxk9x0

Please try to recreate the issue on https://codesandbox.io so I can investigate what is happening.

Cheers!

**Comment by pryahin** [comment]
I meant that if some element has css animation while he is a child AnimateHeight, then after the animation the element will work onAnimationHeight parent

Try adding this code to the style.css
```css
@keyframes open {
  from {
    opacity: 0;
  }
  to {
    opacity: 1
  }
}

h2 {
  opacity: 0;
  animation: open 1s ease-in-out
}
```

**Comment by Stanko** [comment]
Woah! 😄 
Great catch, I think I know what it is. Let me investigate and I'll get back to you asap.

**Comment by Stanko** [comment]
@pryahin bug fixed in version 2.0.5 🎉 

I just omitted `onAnimationStart` and `onAnimationEnd` from being passed to the element directly.
https://github.com/Stanko/react-animate-height/blob/b491426085705f7f6766c41995561e0ed08ecbad/source/AnimateHeight.js#L18-L30

Thing is that callback names were identical to the native DOM handlers, and they were attached here:

https://github.com/Stanko/react-animate-height/blob/b491426085705f7f6766c41995561e0ed08ecbad/source/AnimateHeight.js#L333-L335

I'm passing all props to the element to allow users to pass any accessibility props directly to the div, but I forgot to omit these two.

Links to the native event handlers:

* https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onanimationstart
* https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onanimationend

I updated the example here:
https://codesandbox.io/s/r47yk1ml84

Cheers!
